pkg/apis/build/v1alpha1/build_types.go

```diff
@@ -66,6 +66,7 @@ type BuildSpec struct {
 	Env       []corev1.EnvVar             `json:"env,omitempty"`
 	Resources corev1.ResourceRequirements `json:"resources,omitempty"`
 	LastBuild *LastBuild                  `json:"lastBuild,omitempty"`
+	Notary    NotaryConfig                `json:"notary,omitempty"`
```

Comment:
Should this be nil-able?

Reply:
It could be, but we'd have to do a nil check on both the notary config and the V1 portion of the notary config.

Reply:
Fair enough. Could that be extracted away from the caller in the `NotaryV1Config()` method?

Reply:
Yea it could be. I think thats preferable.